src/Layers/xrRender/BufferUtils.h

```diff
@@ -78,17 +77,16 @@ class VertexStagingBuffer
 private:
     void Destroy();
 
-    VertexBufferHandle m_DeviceBuffer;
-    HostBufferHandle m_HostBuffer;
-    size_t m_Size{ 0 };
-    u32 m_RefCounter{ 0 };
-    bool m_AllowReadBack{ false }; // specifies whether host will want to have the data back (e.g. skinning code)
+	VertexBufferHandle m_DeviceBuffer{};
+	HostBufferHandle m_HostBuffer{};
```

Comment:
You have tabs here...
![image](https://user-images.githubusercontent.com/2989212/177471486-f94f9b13-3701-460f-86be-6835ac287aed.png)

Reply:
oops